server/controllers/wazuh-elastic.js

```diff
@@ -870,7 +869,7 @@ export class WazuhElasticCtrl {
     try {
       // Check if wazuh sample alerts index exists
```

Comment:
Maybe check if `req.params.index_pattern` exists, because it is required. Reply using ErrorResponse if that parameter is not present.

Review the other endpoints of sample data that require `req.params.index_pattern` param to do the same.

Reply:
Should always `req.params.index_pattern` exists because is part of the route of the endpoint.
If this parameter is not included never can be matched with the route, so it will return a 404 error.

Reply:
👍